packages/suite-data/files/translations/en.json

```
@@ -822,7 +822,7 @@
   "TR_EVM_ADDRESS_IS_CONTRACT": "You're sending funds to a contract address.",
   "TR_EVM_EXPLANATION_DESCRIPTION": "It shares the same address style as Ethereum but has its own unique coins and tokens that can't be used on other networks.",
   "TR_EVM_EXPLANATION_EXCHANGE_DESCRIPTION": "You selected {coin} on the {network} network, but it seems you don't have any {networkSymbol} coins. Did you mean to choose {coin} on another network?",
-  "TR_EVM_EXPLANATION_RECEIVE_DESCRIPTION": "This receive address is only for {network} coins and tokens. If someone sends you crypto from outside the {network} network, you may not receive them.",
+  "TR_EVM_EXPLANATION_RECEIVE_DESCRIPTION": "The network uses the same address format as Ethereum. You'll need ETH to pay transaction fees on this network.",
```

```suggestion
  "TR_EVM_EXPLANATION_RECEIVE_DESCRIPTION": "The network uses the same address format as Ethereum. You'll need {networkName} ETH to pay transaction fees on this network.",
```
How about you will need Base ETH? @jirih-stsh

---

@tomasklim yeah agreed, lets say "Base ETH" / "Optimism ETH" / "Arbitrum ETH"
cc @izmy